### DISCUSSION TITLE
Elixir Client library with complete integration!

### DISCUSSION THREAD

**Discussion by zoedsoupe** [supabase/supabase#17541] [Contribute to Supabase]
Hello! I’m here to announce that I’m building a complete rewrite for the Supabase Elixir client library. I know there are some already implemented, but they seens unmaintained and also doesn’t give a intgrated experience with the Elixir world.

So I started with the supabase storage client firstly and the monorepo can be found here: https://github.com/zoedsoupe/supabase

I would like to know if there’s a possibility to officially get support for my implementation. 

The next integration I would like to do is the Supabase UI for [Phoenix Live View](https://hexdocs.pm/phoenix_live_view) and perhaps integrate the Supabase PostgREST with [Ecto](https://hexdocs.pm/ecto)

What do you think about this project?

**Comment by YuriCodes** [comment]
Hi @zoedsoupe ! We are so excited for this idea, thank you for putting this together. We'd love to invite you to the [SupaSquad](https://supabase.com/open-source/contributing/supasquad) and are looking forward to support this project. We have reached out on Twitter DM's to continue this conversation.

**Reply by zoedsoupe** [reply]
Hello @YuriCodes! Thank you for reaching me out and thanks for validating my request for the contribution. I would love to be part of SupaSquad and really liked that you tried to reach me out on Twitter. However I already deactivated my Twitter/X’s account so I’m not able to answers it. Could you send me the communication on my [e-mail](mailto:zoey.spessanha@zeetech.io) or even on my [telegram](https://t.me/zoedsoupe)?
